src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -322,21 +337,21 @@ describe('containers/pages/NewPlan', () => {
         .first()
         .props() as any).formik.values
     );
-
-    expect(mockedStore.dispatch).toHaveBeenLastCalledWith(addPlanDefinition(payload));
+    const payloadOfInterest = extractPlanRecordResponseFromPlanPayload(payload);
+    expect(mockedStore.dispatch).toHaveBeenLastCalledWith(fetchPlanRecords([payloadOfInterest]));
```

Comment:
Could you explain this change?  Trying to understand the significance of swapping `addPlanDefinition(payload)` with `fetchPlanRecords([payloadOfInterest])`?